README.md

```diff
@@ -48,42 +48,68 @@ Thinking about contributing to RES? Awesome! We just ask that you follow a few s
 
 ##### Top level files and folders
 
-  - `README.md`: YOU ARE HERE, unless you're browsing on GitHub
-  - `changelog.txt`: self-explanatory
-  - `gulpfile.babel.js`: build script
-  - `package.json`: package info, dependencies
+  - `assets/`: RES logo source
+  - `chrome/`: Chrome-specific RES files
+  - `dist/`: build output
+  - `edge/`: Microsoft Edge-specific RES files
+  - `examples/`: example code for new hosts/modules
+  - `firefox/`: Firefox-specific RES files
   - `lib/`: all RES code
   - `lib/core/`: core RES code
+  - `lib/css/`: RES css
+  - `lib/environment/`: RES environment code
+  - `lib/images/`: RES images
   - `lib/modules/`: RES modules
-  - `lib/vendor/`: RES vendor libraries
-  - `chrome/`: Chrome-specific RES files
-  - `firefox/`: Firefox-specific RES files
+  - `lib/templates/`: RES templates
+  - `lib/utils/`: RES utilities
+  - `lib/utils/__tests__/`: unit tests
```

Comment:
`__tests__` could be any subdirectory (hence the `**/__tests__`), I don't think we need to point it out in `lib/utils`.

I think either keep the `**/__tests__` or have a note below this list, along the lines of  "unit tests can be in any `__tests__` subdirectory"